services/web/server/src/simcore_service_webserver/garbage_collector_core.py

```diff
@@ -362,7 +366,7 @@ async def _remove_single_orphaned_service(
             except (UserNotFoundError, ValueError):
                 user_role = None
 
-            project_uuid = currently_opened_projects_node_ids[service_uuid]
+            project_uuid = interactive_service["project_id"]
```

Comment:
so in  ``interactive_service`` which is an argument of the function, the key "project_id" is required , right? This is guaranteed. Perhaps we have some TypeDict to express that in the annotation?

Reply:
The director-v2 actually guarantees that that all the fields are present inside the. If you look at the `DynamicServiceGet` model, those fields are mandatory.